### PR TITLE
Adding Status Bar

### PR DESCRIPTION
**Pull request by Andreymazepas**
I just finished this for xp.css, figured I would make a PR here too. This adds a status bar to the bottom of the window.

image

### CODE REVIEW

File: docs/index.html.ejs [hunk 1]
```diff
@@ -635,6 +635,48 @@
       </div>
     </section>
 
+    <section class="component">
+      <h4 id="status-bar">Status Bar</h4>
```
Comment (jdan) [inline docs/index.html.ejs:639]: Wanna link to this in the TreeView on the left side as well?

File: docs/index.html.ejs [hunk 2]
```diff
@@ -635,6 +635,48 @@
...
+          <div class="window-body">
+        <p> There are just so many possibilities:</p>
+        <ul>
+            <li>A Task Manager</li>
+            <li>A Notepad</li>
+            <li>Or even a File Explorer!</li>
+        </ul>
+          </div>
+          <div class="status-bar">
+            <p class="status-bar-field">Press F1 for help</p>
```
Comment (jdan) [inline docs/index.html.ejs:670]: Why `<p>` out of curiosity?

**Review by jdan — COMMENTED** [review]
Hey this looks really handsome, thanks! Two comments inline

File: docs/index.html.ejs (hunk 1 above)
Comment (Andreymazepas) [reply docs/index.html.ejs:640]: Oh I missed that, just fixed it.

File: docs/index.html.ejs (hunk 2 above)
Comment (Andreymazepas) [reply docs/index.html.ejs:671]: Well you could go with a div, no problem. I just thought it would be better as it is text only.

**Pull request merged**